src/components/Lightbox/index.vue

```diff
@@ -0,0 +1,84 @@
+<template>
```

Comment:
This was taken out of Messaging, right?
Not sure if we'll need this (in the same way) elsewhere (where we have Bootstrap support)

Reply:
Not from messaging; but was built like so because messaging doesn't use bootstrap and I wanted these previews to look the same on all apps.